src/components/image_cropper.tsx

```diff
@@ -13,6 +12,8 @@ if ('default' in AvatarEditor) {
     AvatarEditor = AvatarEditor.default;
 }
 
+const debounce = require('lodash.debounce');
```

Comment:
Why do we need to import it like this?

Reply:
The normal import wasn't working, but I have figured at the correct one